ddtrace/span.py

```diff
@@ -359,6 +365,18 @@ def set_tags(self, tags):
             for k, v in iter(tags.items()):
                 self.set_tag(k, v)
 
+    @removed_property(
+        message="Use getters and setters instead of accessing Span.meta directly. \
```

Comment:
Full deprecation warning is :`Reading the 'Span.meta' property is deprecated and will be removed in version '1.0.0': Use getters and setters instead of accessing Span.meta directly. Ex: Span.set_tag(k, v), Span.set_tags(), Span.get_tag(k), Span.get_tags()`

Reply:
I would roll the example into the recommended changes to be made:

```
Use Span.set_tag(), Span.set_tags(), Span.get_tag(k), or Span.get_tags() instead.
```